CONTRIBUTING.md

```diff
@@ -1,87 +1,15 @@
 Contributing to the Linter
 ==========================
 
-[![Build Status](https://github.com/dart-lang/linter/workflows/linter/badge.svg)](https://github.com/dart-lang/linter/actions)
-![GitHub contributors](https://img.shields.io/github/contributors/dart-lang/linter.svg)
+## [Repository has moved][]
```

Comment:
I don't see a target for this link. I'm guessing that you didn't intend for it to be a link.

Reply:
Quite right. I just copied what @parlough did for the README but perhaps that's a mistake too?

Reply:
Fixed.